### PR TITLE
docs: Add minimum and recommended specs for Unleash and for Database

### PR DESCRIPTION
**Pull request by chriswk**
What the title says, adds a subchapter for specs for Unleash, and a subchapter for specs for the database

### CODE REVIEW

File: website/docs/using-unleash/deploy/configuring-unleash.md [hunk 1]
```diff
@@ -482,6 +498,30 @@ same SSL modes that they support. As of version 8 (released on February 25th
 reason, Unleash cannot support all of Postgres' SSL modes. Specifically, Unleash **does not support** `sslmode=prefer`.
 Instead you must know whether you require an SSL connection ahead of time.
 
+### Recommended Postgres spec
+
+#### Minimum
+
+For Postgresql we recommend running with at least 1 CPU and at least 1Gi of memory, as well as at least 5Gi of storage.
```
Comment (nunogois) [inline website/docs/using-unleash/deploy/configuring-unleash.md:505]:
```suggestion
For PostgreSQL we recommend running with at least 1 CPU and at least 1Gi of memory, as well as at least 5Gi of storage.
```

File: website/docs/using-unleash/deploy/configuring-unleash.md [hunk 2]
```diff
@@ -482,6 +498,30 @@ same SSL modes that they support. As of version 8 (released on February 25th
 reason, Unleash cannot support all of Postgres' SSL modes. Specifically, Unleash **does not support** `sslmode=prefer`.
 Instead you must know whether you require an SSL connection ahead of time.
 
+### Recommended Postgres spec
```
Comment (nunogois) [inline website/docs/using-unleash/deploy/configuring-unleash.md:501]:
```suggestion
### Recommended PostgreSQL spec
```

**Review by gastonfournier — APPROVED** [review]
LGTM!

**Review by nunogois — APPROVED** [review]
LG

**Pull request merged**